pkg/agent/agent.go

```diff
@@ -454,11 +485,21 @@ func (i *Initializer) initNodeLocalConfig() error {
 		return err
 	}
 
-	i.nodeConfig = &types.NodeConfig{Name: nodeName, PodCIDR: localSubnet}
+	localAddr, dev, err := util.GetDefaultLocalNodeAddr()
+	if err != nil {
+		return fmt.Errorf("initNodeLocalConfig %v", err)
+	}
+
+	servRtTable := &types.ServiceRtTableConfig{Idx: 254, Name: "main"}
+	if i.podEncapMode.SupportsNoEncap() {
+		servRtTable.Idx = 300
+		servRtTable.Name = "Antrea-service"
+	}
+	i.nodeConfig = &types.NodeConfig{Name: nodeName, PodCIDR: localSubnet, NodeIPAddr: localAddr, NodeDefaultDev: dev.Attrs().Name, PodEncapMode: i.podEncapMode, ServiceCIDR: i.serviceCIDR, ServiceRtTable: servRtTable}
```

Comment:
I need to look into these, but wonder: 1) do we put all them in NodeConfig, esp. if they are not shared by other NodeConfig consumers. Now NodeConfig just has a small set of Node info. 2) do we need to initialize and also save (all) them in agentInitializer. For example, are some of them used by only routeClient?

Reply:
routable is now moved out of NodeConfig